splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/UpdateOperationIT.java

```diff
@@ -684,4 +685,21 @@ private int doTestUpdateMultiColumnOneSubSyntax(String outerWhere) throws Except
         }
     }
 
+    @Test
+    public void testUpdateWithTableAlias() throws Exception {
+        new TableCreator(methodWatcher.getOrCreateConnection())
+                .withCreate("create table alias1 (a1 int, b1 int, c1 int)")
+                .create();
+
+        new TableCreator(methodWatcher.getOrCreateConnection())
+                .withCreate("create table alias2 (a2 int, b2 int, c2 int)")
+                .create();
+
+        Assert.assertTrue("expected result having Subquery",
+                SpliceUnitTest.getExplainMessage(3,"explain update alias1 X set a1 = (select y.a2 from alias2 y where b1=y.b2)",methodWatcher).contains("Subquery"));
+        Assert.assertTrue("expected result having BroadcastJoin",
+            SpliceUnitTest.getExplainMessage(3,"explain update alias1 X set (a1) = (select y.a2 from alias2 y where b1=y.b2)",methodWatcher).contains("BroadcastJoin"));
+
```

Comment:
The difference between the two update statements is a pair of parenthesis around column a1. They are semantically equivalent, but why they have very different plans?

Reply:
The columns braced by parenthesis are used for updating multiple columns, which is optimized as join update. 
The single column without braced parenthesis update from a subquery.